resources/qml/Menus/MaterialBrandMenu.qml

```diff
@@ -94,16 +94,21 @@ Cura.MenuItem
 
         x: parent.width - UM.Theme.getSize("default_lining").width
         y: {
-            // Checks if popup is more than halfway down the screen AND further than 400 down (this avoids popup going off the top of screen)
-            // If it is then the popup will push up instead of down
-            // This fixes the popups appearing bellow the bottom of the screen.
+            var popupHeight = materialTypesModel.material_types.count * UM.Theme.getSize("menu").height
```

Comment:
Does this take spacing into account between the items?

Reply:
The spacing is set to 0 for the Columns below. I do see the alignment is slightly off though

Reply:
I adjusted the alignment and now they line up perfectly with their parent.

Reply:
I adjusted the alignment and now they line up perfectly with their parent.